lib/configuration/applyConfiguration.js

```diff
@@ -122,10 +122,7 @@ function applyConfiguration(config) {
   return resolvedConfig;
 }
 
-module.exports = {
-  applyConfiguration,
-  resolveConfig,
-  _utils: {
-    DEFAULT_CONFIG,
-  },
-};
+applyConfiguration.resolveConfig = resolveConfig;
+applyConfiguration.DEFAULT_CONFIG = DEFAULT_CONFIG;
```

Comment:
I see your point about collisions with builtin function properties. What we did in other files was underscore dangling so it never happens:

```js
applyConfiguration._resolveConfig = resolveConfig;
applyConfiguration._DEFAULT_CONFIG = DEFAULT_CONFIG;
```

I wonder whether it's needed or not. Looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function I don't think we're likely to hit any of the prototype properties 🤔 Maybe we could remove the dangling from other files then (if you agree, we could do it in a separate PR and then we can remove https://github.com/apiaryio/dredd/blob/master/.eslintrc.js#L22-L25 as well).